mne/minimum_norm/tests/test_resolution_matrix.py

```diff
@@ -0,0 +1,111 @@
+fname_src_3 = op.join(data_path, 'subjects', 'sample', 'bem',
+                      'sample-oct-4-src.fif')
+fname_stc = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-meg')
+fname_vol = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-grad-vol-7-fwd-sensmap-vol.w')
+fname_vsrc = op.join(data_path, 'MEG', 'sample',
+                     'sample_audvis_trunc-meg-vol-7-fwd.fif')
+fname_inv_vol = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc-meg-vol-7-meg-inv.fif')
+rng = np.random.RandomState(0)
+
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
+fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-cov.fif')
+
+# get functions for resolution matrix, metrics etc.
+
+
+@testing.requires_testing_data
+def test_resolution_matrix():
+    """Test whether MNE's resolution matrix is symmetric."""
+    # read forward solution
+    forward = mne.read_forward_solution(fname_fwd)
+
+    # forward operator with fixed source orientations
+    forward_fxd = mne.convert_forward_solution(forward, surf_ori=True,
+                                               force_fixed=True)
+
+    # noise covariance matrix
+    noise_cov = mne.read_cov(fname_cov)
+
+    # evoked data for info
+    evoked = mne.read_evokeds(fname_evoked, 0)
+
+    # make inverse operator from forward solution
+    # free source orientation
+    inverse_operator = mne.minimum_norm.make_inverse_operator(
+        info=evoked.info, forward=forward, noise_cov=noise_cov, loose=1.,
+        depth=None)
+    # fixed source orientation
+    inverse_operator_fxd = mne.minimum_norm.make_inverse_operator(
+        info=evoked.info, forward=forward, noise_cov=noise_cov, loose=0.,
+        depth=None, fixed=True)
+
+    # regularisation parameter based on SNR
+    snr = 3.0
+    lambda2 = 1.0 / snr ** 2
+
+    # resolution matrices for free source orientation
+
+    # compute resolution matrix for MNE with free source orientations
+    rm_mne_free = make_resolution_matrix(forward, inverse_operator,
+                                         method='MNE', lambda2=lambda2)
+
+    # resolution matrices for fixed source orientation
+
+    # compute resolution matrix for MNE
+    rm_mne = make_resolution_matrix(forward_fxd, inverse_operator_fxd,
+                                    method='MNE', lambda2=lambda2)
+
+    # compute resolution matrix for sLORETA
+    rm_lor = make_resolution_matrix(forward_fxd, inverse_operator_fxd,
+                                    method='sLORETA', lambda2=lambda2)
+
+    # rectify resolution matrix for sLORETA before determining maxima
+    rm_lor_abs = np.abs(rm_lor)
+
+    # get maxima per column
+    maxidxs = rm_lor_abs.argmax(axis=0)
+
+    # create array with the expected stepwise increase in maximum indices
+    goodidxs = np.arange(0, len(maxidxs), 1)
+
+    # Tests
+    # Does sLORETA have zero DLE for columns?
```

Comment:
```suggestion
    # Does sLORETA have zero localization error for columns?
```

avoid acronyms as they are harder to understand for new comers